webapp/packages/core-blocks/src/Containers/ColoredContainer.tsx

```diff
@@ -6,12 +6,18 @@
  * you may not use this file except in compliance with the License.
  */
 
+import { forwardRef, Ref } from 'react';
+
 import type { IContainerProps } from './IContainerProps';
 
-export const ColoredContainer: React.FC<IContainerProps> = function ColoredContainer({ children, className }) {
+interface Props extends IContainerProps {
+  ref?: Ref<HTMLDivElement>;
+}
```

Comment:
think this property is invalid, because of injecting rest as the second argument after properties